blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -104,5 +104,9 @@ Changes
 Fixes
 =====
 
+- Fix serialization issue that might occur in distributed queries that
+  contain a standalone reference and partitioned window function call in
```

Comment:
Not sure if `standalone reference` is meaningful for end-users who lack the context of how the implementation works.

Reply:
will keep only `window function with partition by clause`